scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -122,6 +122,8 @@ namespace <%= namespace %> {
             State = new CartState(client);
             Client = client;
 
+            ExternalBrowserCheckout = new DesktopCheckout(this, client);
```

Comment:
I think it makes sense to have this always available, as Application.OpenUrl can be attempted by the Unity player on every platform (I believe), however the name should definitely be changed from Desktop, that was just my first guess at the right name, it should probably reference that it will be an ExternalBrowserCheckout